package.json

```diff
@@ -21,7 +21,9 @@
     "build-js": "node tools/build-js.js",
     "minify-js": "node tools/minify-js.js",
     "watch-js": "watchify -d src/js/app.js -o build/gen/app.js -v",
-    "lint": "eslint src/js/** tools/**",
+    "lint": "npm run lint-js",
```

Comment:
Since stylelint is pretty fast, why not make this run both lint-js and lint-css? With concurrently there might be nearly no overhead.